src/Mutation/FilteredCollection.php

```diff
@@ -4,14 +4,14 @@
 
 use Softonic\GraphQL\Mutation\Traits\MutationObjectHandler;
 
-class FilteredCollection implements MutationObject, \JsonSerializable
+class FilteredCollection implements MutationObject, \IteratorAggregate, \JsonSerializable
 {
     use MutationObjectHandler;
 
     /**
      * @var array<Item>
      */
-    protected $arguments = [];
+    public $arguments = [];
```

Comment:
Why public?

Reply:
It's the only way the Iterator from FilteredCollection works. It does not work with a getter.

Reply:
What exactly is not working?